### PR TITLE
Broadcast players tab

### PR DESCRIPTION
**Pull request by julien4215**
[broadcast_players.webm](https://github.com/user-attachments/assets/a430577e-b537-482d-8a27-aa89984accd6)

Tablet :

Screenshot_1733842417

### CODE REVIEW

File: lib/src/model/broadcast/broadcast_providers.dart [hunk 1]
```diff
@@ -54,6 +55,50 @@ Future<BroadcastTournament> broadcastTournament(
...
+    BroadcastTournamentId tournamentId,
+  ) async {
+    final players = ref.withClient(
+      (client) => BroadcastRepository(client).getPlayers(tournamentId),
+    );
+
+    return players;
+  }
+
+  void sort(BroadcastPlayersSortingTypes sortingType, [bool reverse = false]) {
```
Comment (veloce) [inline lib/src/model/broadcast/broadcast_providers.dart:73]: Looks like this function is not used since the sorting is done in the widget state.

**Review by veloce — COMMENTED** [review]
Nice work! thanks

File: lib/src/model/broadcast/broadcast_providers.dart (hunk 1 above)
Comment (julien4215) [reply lib/src/model/broadcast/broadcast_providers.dart]: Oops I forgot to remove it when moving the sort in the widget

**Comment by ijm8710** [comment]
Very cool Julian. Just making sure you're aware, some tournaments have a teams section as well.

Random example: 

Slovenian Team Championship 2024

**Pull request merged**